### PR TITLE
ref(ui): Allow alertMessages messages to be nodes

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/alertMessage.jsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default class AlertMessage extends React.PureComponent {
   static propTypes = {
     alert: PropTypes.shape({
       id: PropTypes.string,
-      message: PropTypes.string.isRequired,
+      message: PropTypes.node.isRequired,
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/alertMessage.jsx]: Should this be node or string?

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/alertMessage.jsx]: node, so you can add things like links.

Comment (markstory) [reply src/sentry/static/sentry/app/components/alertMessage.jsx]: But if it was a string before, could this change cause prop type warnings in other uses?

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/alertMessage.jsx]: `node` encompasses string

**Review by markstory — APPROVED** [review]

**Pull request merged**